src/app/modules/backbone/components/header/header.component.html

```diff
@@ -0,0 +1,20 @@
+<nav class="main-nav">
+    <header class="brand">
+        <img src="/favicon.ico"/>
+        <span>DAK</span>
+    </header>
+    <div class="burger-menu-wrapper">
+        <app-burger-menu [onBurgerMenuClick]="onBurgerMenuClick"></app-burger-menu>
```

Comment:
we should always prefer to define @Output in the child component and handle the event here. It is bad practice to pass function references